### PR TITLE
Add meta-mender overview.

### PR DESCRIPTION
**Pull request by pasinskim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
-# yocto
-Yocto layer for Mender client
+# meta-mender
+
+This README file contains information about the Yocto layer designed for building Mender client application as a part of the Yocto image.
```
Comment (eysteins) [inline README.md]: "...building *the* Mender client application..."

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+1. Overview
+2. What is Mender
+3. Using Mender
+
+
+
+Overview
+========
+
+This layer contains all the needed recipes to build Mender Go binary as a part of the Yocto image. Currently we are supporting
```
Comment (eysteins) [inline README.md]: ..to build *the* Mender Go binary as part of *a* Yocto image."

Comment (eysteins) [inline README.md]: "It currently supports cross-compiling Mender"...

File: README.md [hunk 3]
```diff
@@ -1,2 +1,49 @@
...
+What is Mender
+==============
+
+Detailed Mender description is provided in [mender repository](https://github.com/mendersoftware/mender).
+
+
+Using Mender
+============
+
+To quickly get started, Mender can be tested using the qemu emulator. Detailed instructions how to build a Yocto image that can be run and tested in qemu are provided in the [meta-mender-qemu repository](https://github.com/mendersoftware/meta-mender-qemu).
```
Comment (eysteins) [inline README.md]: "Detailed instructions *for* how.."

File: README.md [hunk 4]
```diff
@@ -1,2 +1,49 @@
...
+This layer contains all the needed recipes to build Mender Go binary as a part of the Yocto image. Currently we are supporting
+cross-compiling Mender for ARM devices using Go 1.4 and 1.5.
+
+As Mender is a framework not just a standalone application it requires the bootloader and partition layout set up in a specific way. That's why it is recommended to
+use Yocto for building a complete image containing all the needed dependencies and configuration.
+
+Detailed instructions and recipes needed for building a self-containing image can be found in [meta-mender-qemu repository](https://github.com/mendersoftware/meta-mender-qemu).
+
+
+What is Mender
```
Comment (eysteins) [inline README.md]: Perhaps move this to top?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,49 @@
...
+==============
+
+Detailed Mender description is provided in [mender repository](https://github.com/mendersoftware/mender).
```
Comment (eysteins) [inline README.md]: "A detailed description of Mender is provided in the [mender ..."

**Comment by pasinskim** [comment]
@eysteins thanks for reviewing. All should be fixed now.

**Pull request merged**